# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/*/epplets-extra16/e-mount.patch
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- E-Mount_0.3/Makefile.orig-rock-maint	2003-08-29 09:31:58.000000000 +0200
+++ E-Mount_0.3/Makefile	2003-08-29 09:32:49.000000000 +0200
@@ -2,13 +2,13 @@
 
 IMLIB_CFLAGS = `imlib-config --cflags`
 IMLIB_LIBS = `imlib-config --libs`
-GCCOPTS = -MMD -Wall
+GCCOPTS = -MMD -Wall $(CFLAGS)
 GCC = gcc
 
 all : E-Mount.epplet
 
 E-Mount.epplet : E-Mount.c
-	${GCC} ${GCCOPTS}  E-Mount.c -o E-Mount.epplet -lepplet ${IMLIB_LIBS} ${IMLIB_CFLAGS}
+	${GCC} ${GCCOPTS}  E-Mount.c -o E-Mount.epplet -L$(E16)/lib -lepplet ${IMLIB_LIBS} ${IMLIB_CFLAGS}
 
 install : E-Mount.epplet
 	install -m 755 E-Mount.epplet ${EBIN}/E-Mount.epplet
